src/app/about/about.component.ts

```diff
@@ -11,8 +12,16 @@ export class AboutComponent implements OnInit {
 
   version: string = environment.version;
 
-  constructor() { }
+  constructor(private dataService: DataService) { }
 
   ngOnInit() { }
 
+  callUrl() {
```

Comment:
this string should be injected into the component like a service. I think you do this with an OpaqueToken although I vaguely remember that being deprecated. For now, I would be happy with a constants class that has static properties on it that we stick these types of values on.